modules/vba_documents/lib/vba_documents/upload_validator.rb

```diff
@@ -71,10 +71,15 @@ def validate_names(first, last)
       end
     end
 
-    def validate_line_of_business(lob)
-      return if lob.to_s.empty?
+    def validate_line_of_business(lob, submission_version)
+      return if lob.to_s.empty? && !(submission_version && submission_version >= 2)
 
-      unless VALID_LOB.keys.include?(lob)
+      if lob.to_s.blank? && Settings.vba_documents.v2_enabled && submission_version >= 2
```

Comment:
Why do we need the settings check here?  We couldn't have added the value of two unless we got to the v2 route or am I missing something?